### PR TITLE
Add student page to the directory

### PR DESCRIPTION
**Pull request by ZoeWard15891**
This pull request is to add my page to the directory, please review @githubteacher

### CODE REVIEW

File: _pages/ZoeWard15891.yaml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+githubHandle: ZoeWard15891
+pageUrl: ZoeWard15891.github.io/
```
Comment (hectorsector) [inline _pages/ZoeWard15891.yaml]: @ZoeWard15891 just remove the `/` and you should be all set

**Comment by hectorsector** [comment]
You're all set @ZoeWard15891 we made some changes to the test so that's why it's having so much trouble. I'll go ahead and merge this with your 👍

**Comment by ZoeWard15891** [comment]
Yes please, I've been feeling that I'm missing something obvious, apparently I wasn't.

**Comment by hectorsector** [comment]
@ZoeWard15891 I'm sorry about that! You're all set and your site is listed in our directory 🎉 

https://services.github.com/on-demand/github-desktop/16/

**Pull request merged**